test/integration/fake_upstream.cc

```diff
@@ -83,10 +83,10 @@ void FakeStream::encode100ContinueHeaders(const Http::ResponseHeaderMap& headers
 void FakeStream::encodeHeaders(const Http::HeaderMap& headers, bool end_stream) {
   std::shared_ptr<Http::ResponseHeaderMap> headers_copy(
       Http::createHeaderMap<Http::ResponseHeaderMapImpl>(headers));
-  if (add_served_by_header_) {
+  if (add_served_by_header_)
```

Comment:
What is this style change? I thought we had mandatory braces for single line `if`. I think Heartbleed or similar was caused by not doing this.

Reply:
No, I'm trying to prove clang-tidy will fail on this, and revert later.